test/prototype_transforms_dispatcher_infos.py

```diff
@@ -50,6 +50,13 @@ def maybe_skip(self, *, test_name, args_kwargs, device):
             features.BoundingBox: F.resize_bounding_box,
             features.Mask: F.resize_mask,
         },
+        skips=[
```

Comment:
These skips currently don't do anything since we don't have integer sizes, but will be necessary as soon that happens.